### DISCUSSION TITLE
App won't start using conda env

### DISCUSSION THREAD

**Discussion by tmostak** [h2oai/h2o-llmstudio#416] [Q&A]
I'm having issues starting the server when using a conda environment on a paperspace machine.

I've done this on other machines with no issue, so not sure what's the difference.

I've run

```
conda create -y -n h2o_llm2 python=3.10.9
conda activate h2o_llm2
python3 -m pip install -r requirements.txt
H2O_WAVE_APP_ADDRESS=http://127.0.0.1:8756 H2O_WAVE_MAX_REQUEST_SIZE=25MB H2O_WAVE_NO_LOG=true H2O_WAVE_PRIVATE_DIR=/download/@output/download wave run --no-reload app
```

And I get the following:

Could not connect to Wave server. Please start the Wave server (waved or waved.exe) prior to running any app.

If I open a Python shell I can import wave, transformers, etc. so it seems like the neccessary libraries are there.

**Comment by pascal-pfeiffer** [comment]
The following works on a fresh machine and I wasn't able to reproduce your issue.
Please check for any potential firewall settings that would block the app.

```sh
conda create -n llmstudio python=3.10
conda activate llmstudio
conda install -c "nvidia/label/cuda-11.8.0" cuda-toolkit
make setup
make llmstudio
```
